server/db/models/product.js

```diff
@@ -0,0 +1,61 @@
+const Sequelize = require('sequelize');
+const db = require('../db');
+
+const Product = db.define('product', {
+	title: {
+		type: Sequelize.STRING,
+		allowNull: false
+	},
+	description: {
+		type: Sequelize.TEXT
+	},
+	imageUrl: {
+		type: Sequelize.ARRAY(Sequelize.STRING),
+		defaultValue: [ 'http://products.ideadunes.com/assets/images/default_product.jpg' ]
+	},
+	price: {
+		//Unit type is cents
+		type: Sequelize.INTEGER,
+		allowNull: false,
+		validate: {
+			min: 0
+		}
+	},
+	quantity: {
+		type: Sequelize.INTEGER,
+		allowNull: false,
+		validate: {
+			min: 0
+		}
+	},
+	isActive: {
```

Comment:
I think this should be false by default. When admin adds a product, it should be passive. Then admin can click publish button to make it active. If you think this will take time now, we can change it later.